README.md

```diff
@@ -811,34 +811,42 @@ build type).  The static libraries will not be built (since
 
   If `ON`, OpenMP is used if it is available.  Default: `OFF`.
 
-  UMFPACK, CHOLMOD, SPQR, and GraphBLAS will be slow if OpenMP is not used.
+  GraphBLAS, LAGraph, and ParU will be slow if OpenMP is not used.
```

Comment:
IIUC, `CHOLMOD_CUDA` also uses OpenMP (directly - not just via an accelerated BLAS or LAPACK implementation).

Reply:
Yes, but it's modest impact to the performance of CHOLMOD (either the GPU or CPU).  CHOLMOD gets most of its parallel performance inside the BLAS/LAPACK.

I will update the readme to clarify this point.

Reply:
see https://github.com/DrTimothyAldenDavis/SuiteSparse/commit/b9a18492097424e169f5ec786799d609af7ab006